extensions/amp-ad-network-oblivki-impl/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'cathyxz'}, {name: 'jeffkaufman'}, {name: 'vvsoblivki'}],
```

Comment:
@rsimha @rcebulko do you think this can be the cause that this PR got sent to wrong reviewers?

Reply:
Definitely. As far as the Owners Bot can tell from this config, Cathy and Jeff are still the owners. I'd recommend putting `ampproject/wg-ads` here in their place, and if Jeff shouldn't be requested, add `requestReviews: false` to his owner config. See [`ads/OWNERS`](https://github.com/ampproject/amphtml/blob/master/ads/OWNERS) for an example of where this is already done.

Reply:
no, these changes were made last

Reply:
Correction: I didn't realize this was a new Owners file. Based on the files in this directory, Jeff was added because (at the time the PR was opened), he was an owner in `ads/OWNERS`; he has since been marked `requestReviews: false`. Cathy is a member of `wg/ui-and-a11y`, which owns `extensions/` and so was added to approve the new extension directory.